### PR TITLE
kubeadm: remove some Docker cgroup driver notes

### PR DESCRIPTION
**Pull request by neolit123**
Link to existing guide pages instead.
The link also has notes about other container runtimes and not only Docker.

/kind cleanup
/sig cluster-lifecycle

### CODE REVIEW

File: content/en/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md [hunk 1]
```diff
@@ -89,23 +89,10 @@ If you notice that `kubeadm init` hangs after printing out the following line:
...
-
-  1. Install Docker again following instructions
-     [here](/docs/setup/production-environment/container-runtimes/#docker).
-
-  1. Change the kubelet config to match the Docker cgroup driver manually, you can refer to
-     [Configure cgroup driver used by kubelet on control-plane node](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#configure-cgroup-driver-used-by-kubelet-on-control-plane-node)
-
-- control plane Docker containers are crashlooping or hanging. You can check this by running `docker ps` and investigating each container by running `docker logs`.
+- the cgroup driver of the container runtime differs from that of the kubelet. To understand how to
+configure it properly see [Configuring a cgroup driver](https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver/).
```
Comment (kbhawkey) [inline content/en/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md:93]: Hi @neolit123 .
Would you change the link to:
```
 [Configuring a cgroup driver](/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver/).
```

Comment (neolit123) [reply content/en/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md]: @kbhawkey i have updated the link.

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-28359--kubernetes-io-main-staging.netlify.app/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm/#kubeadm-blocks-waiting-for-control-plane-during-installation

File: content/en/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md [hunk 2]
```diff
@@ -89,23 +89,10 @@ If you notice that `kubeadm init` hangs after printing out the following line:
...
-     [here](/docs/setup/production-environment/container-runtimes/#docker).
-
-  1. Change the kubelet config to match the Docker cgroup driver manually, you can refer to
-     [Configure cgroup driver used by kubelet on control-plane node](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#configure-cgroup-driver-used-by-kubelet-on-control-plane-node)
-
-- control plane Docker containers are crashlooping or hanging. You can check this by running `docker ps` and investigating each container by running `docker logs`.
+- the cgroup driver of the container runtime differs from that of the kubelet. To understand how to
+configure it properly see [Configuring a cgroup driver](/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver/).
+- control plane containers are crashlooping or hanging. You can check this by running `docker ps`
+and investigating each container by running `docker logs`. Similar for other container runtimes.
```
Comment (kbhawkey) [inline content/en/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md:95]: :wave: @neolit123 .
Thanks for updating the link. What do you think about adding a bit more detail in the last sentence?
For other container runtimes, run ...

Comment (neolit123) [reply content/en/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md:95]: i think it's is better to replace the last sentence with:

```
For other container runtime see [Debugging Kubernetes nodes with crictl](/docs/tasks/debug-application-cluster/crictl/).
```

https://kubernetes.io/docs/tasks/debug-application-cluster/crictl/

Comment (neolit123) [reply content/en/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md]: updated.

i think it's better to link to the existing crictl page, which is the tool for non-docker container runtimes. has similar commands `ps`, `logs`.

**Comment by neolit123** [comment]
ping @kbhawkey for re-review / lgtm.

**Comment by kbhawkey** [comment]
Thanks @neolit123 .
/lgtm

**Pull request merged**